L1Trigger/Phase2L1ParticleFlow/plugins/L1TPFProducer.cc

```diff
@@ -156,7 +195,7 @@ L1TPFProducer::produce(edm::Event& iEvent, const edm::EventSetup& iSetup) {
     iEvent.put(l1regions_.fetchTracks(/*ptmin=*/0.0, /*fromPV=*/false), "TK");
 
     // Then do the vertexing, and save it out
-    float z0;
+    // float z0;
     l1pfalgo_->doVertexing(l1regions_.regions(), vtxAlgo_, z0);
```

Comment:
considering that now the doVertexing only does the track<->PV association, maybe it should be renamed to associateTracksToPV.
also, the z0 argument should become a `const float &`.